public/perc_df.json

```diff
@@ -0,0 +1 @@
"perc\":0.1931},{\"wyday\":326,\"n_sites\":308,\"n_sites_total\":1559,\"perc\":0.1976},{\"wyday\":327,\"n_sites\":315,\"n_sites_total\":1558,\"perc\":0.2022},{\"wyday\":328,\"n_sites\":322,\"n_sites_total\":1558,\"perc\":0.2067},{\"wyday\":329,\"n_sites\":333,\"n_sites_total\":1558,\"perc\":0.2137},{\"wyday\":330,\"n_sites\":331,\"n_sites_total\":1557,\"perc\":0.2126},{\"wyday\":331,\"n_sites\":336,\"n_sites_total\":1554,\"perc\":0.2162},{\"wyday\":332,\"n_sites\":342,\"n_sites_total\":1554,\"perc\":0.2201},{\"wyday\":333,\"n_sites\":347,\"n_sites_total\":1551,\"perc\":0.2237},{\"wyday\":334,\"n_sites\":339,\"n_sites_total\":1550,\"perc\":0.2187},{\"wyday\":335,\"n_sites\":334,\"n_sites_total\":1548,\"perc\":0.2158},{\"wyday\":336,\"n_sites\":332,\"n_sites_total\":1546,\"perc\":0.2147},{\"wyday\":337,\"n_sites\":342,\"n_sites_total\":1544,\"perc\":0.2215},{\"wyday\":338,\"n_sites\":343,\"n_sites_total\":1542,\"perc\":0.2224},{\"wyday\":339,\"n_sites\":340,\"n_sites_total\":1541,\"perc\":0.2206},{\"wyday\":340,\"n_sites\":351,\"n_sites_total\":1539,\"perc\":0.2281},{\"wyday\":341,\"n_sites\":362,\"n_sites_total\":1537,\"perc\":0.2355},{\"wyday\":342,\"n_sites\":364,\"n_sites_total\":1537,\"perc\":0.2368},{\"wyday\":343,\"n_sites\":350,\"n_sites_total\":1536,\"perc\":0.2279},{\"wyday\":344,\"n_sites\":346,\"n_sites_total\":1535,\"perc\":0.2254},{\"wyday\":345,\"n_sites\":352,\"n_sites_total\":1529,\"perc\":0.2302},{\"wyday\":346,\"n_sites\":344,\"n_sites_total\":1529,\"perc\":0.225},{\"wyday\":347,\"n_sites\":342,\"n_sites_total\":1529,\"perc\":0.2237},{\"wyday\":348,\"n_sites\":335,\"n_sites_total\":1529,\"perc\":0.2191},{\"wyday\":349,\"n_sites\":332,\"n_sites_total\":1527,\"perc\":0.2174},{\"wyday\":350,\"n_sites\":342,\"n_sites_total\":1526,\"perc\":0.2241},{\"wyday\":351,\"n_sites\":347,\"n_sites_total\":1525,\"perc\":0.2275},{\"wyday\":352,\"n_sites\":344,\"n_sites_total\":1521,\"perc\":0.2262},{\"wyday\":353,\"n_sites\":342,\"n_sites_total\":1519,\"perc\":0.2251},{\"wyday\":354,\"n_sites\":347,\"n_sites_total\":1518,\"perc\":0.2286},{\"wyday\":355,\"n_sites\":356,\"n_sites_total\":1516,\"perc\":0.2348},{\"wyday\":356,\"n_sites\":359,\"n_sites_total\":1513,\"perc\":0.2373},{\"wyday\":357,\"n_sites\":371,\"n_sites_total\":1512,\"perc\":0.2454},{\"wyday\":358,\"n_sites\":361,\"n_sites_total\":1512,\"perc\":0.2388},{\"wyday\":359,\"n_sites\":355,\"n_sites_total\":1508,\"perc\":0.2354},{\"wyday\":360,\"n_sites\":361,\"n_sites_total\":1508,\"perc\":0.2394},{\"wyday\":361,\"n_sites\":367,\"n_sites_total\":1506,\"perc\":0.2437},{\"wyday\":362,\"n_sites\":367,\"n_sites_total\":1505,\"perc\":0.2439},{\"wyday\":363,\"n_sites\":365,\"n_sites_total\":1504,\"perc\":0.2427},{\"wyday\":364,\"n_sites\":370,\"n_sites_total\":1503,\"perc\":0.2462},{\"wyday\":365,\"n_sites\":372,\"n_sites_total\":1502,\"perc\":0.2477},{\"wyday\":366,\"n_sites\":344,\"n_sites_total\":1499,\"perc\":0.2295},{\"wyday\":367,\"n_sites\":134,\"n_sites_total\":771,\"perc\":0.1738}]}]"]
```

Comment:
Is this something the pipeline needs to produce and push to S3?

Reply:
Yeah, I am still not able to do it to the public bucket

Reply:
Is the data created via pipeline, but you just can't build to the public bucket?

Reply:
I just added it to the pipeline but it currently exports to the public folder and gets ignored.  https://github.com/USGS-VIZLAB/gw-conditions/pull/33/commits/f535d8cb671beedc6762e6a56b840961d977b23f